db/psql/getAllClients.js

```diff
@@ -0,0 +1,17 @@
+var db = require('../config.js');
+//This function gets all clients from salesperson_client for a specific salesperson
+
+
+var getAllClientsString = "SELECT * FROM client INNER JOIN salesperson_client on (salesperson_client.client_id = client.client_id) WHERE salesperson_client.salesperson_id = $1"
+
+module.exports = function(salesPersonId){
```

Comment:
Can we add a next callback to the exported function, that gets called after the database call and passes in the result?